vercel.json

```diff
@@ -3,6 +3,11 @@
         { "src": "main.py", "use": "@vercel/python" }
     ],
     "routes": [
-        { "src": "/.*/.*/.*", "dest": "main.py" }
-    ]
+        { "src": "/.*/.*/.*", "dest": "main.py" },
+        { "src": "/", "dest": "main.py" }
+    ],
+    "env": {
+        "GITHUB_AUTHORIZATION": "Bearer YOUR-TOKEN",
+        "GITHUB_IS_AUTHORIZED": "False"
+    }
```

Comment:
Maybe just use `.env` file as the default only way of managing the environment variables, so that it's less confusing to users.